logging/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "nodejs-docs-samples-logging",
+  "description": "Node.js samples for Google Cloud Logging.",
+  "version": "0.0.1",
+  "private": true,
+  "license": "Apache Version 2.0",
+  "scripts": {
+    "list": "node list.js",
+    "write": "node write.js",
+    "export": "node export.js"
+  },
+  "dependencies": {
+    "gcloud": "^0.27.0"
+  }
```

Comment:
Lol. I can't move on to anything else until I comment on how much I don't want to add this field.

My stance is that for samples where we actually invite the user to consume and/or deploy (like when there's an `app.yaml` file) then we should specify the `engines` field. e.g. The package.json for `gcloud` should specify `engines`, our MVMs samples should specify `engines`, etc.

Otherwise, for these samples, it's just bloat, something to be maintained and verified (or not). This sample is not going to be installed by NPM or consumed by a deploy or executed by some platform. There isn't anything that's going to programmatically read that field, only eyeballs will see it. I can set `engines` to `0.6.x` and run it with via NPM with Node v5 and NPM doesn't care. If I set it to >=0.12, that implies it only works on >=0.12, which isn't true. I fail to see a benefit that outweighs the cost of maintaining a magic number across dozens of samples.

> It is a strange fate we should suffer so much fear and doubt… over so small a thing.
>
> -- Boromir